### PR TITLE
[minor] damage overlays fix

### PR DESCRIPTION
**Pull request by kit-katz**
## About The Pull Request

fixes an issue where damage overlays overflow and disappear once the damage exceeds the bodypart's health

## Why It's Good For The Game

you cant go wrong with fixing a feature.

## Changelog
:cl:
fix: damage overlays no longer disappear once you deal too much damage
/:cl:

### CODE REVIEW

File: code/modules/surgery/bodyparts/bodyparts.dm
Comment (park66665) [inline code/modules/surgery/bodyparts/bodyparts.dm]:
```suggestion
	if((tbrute != brutestate) || (tburn != burnstate))
```

File: code/modules/surgery/bodyparts/bodyparts.dm
Comment (park66665) [inline code/modules/surgery/bodyparts/bodyparts.dm]:
```suggestion
	var/tbrute = round((min(brute_dam, max_damage) / max_damage) * 3, 1)
	var/tburn = round((min(burn_dam, max_damage) / max_damage) * 3, 1)
```

**Comment by park66665** [comment]
Also, what's up with the changes in human_parts_greyscale.dmi?

**Comment by kit-katz** [comment]
huh odd. that one was already merged in a prior PR. why is it here again

**Comment by park66665** [comment]
If you see no reason why it was included, then remove it from the PR changes.

**Comment by PowerfulBacon** [comment]
why is bodypart damage exceeding the maximum?

**Comment by kit-katz** [comment]
idunno always has been

**Review by park66665 — CHANGES_REQUESTED** [review]
I won't merge this PR if my remaining review isn't properly addressed.

Let me explain since it seems that you don't get it. The way you fixed the bug actually isn't right.

Suppose there is a limb damaged up to approx. 2/3 of `max_damage`. Due to how `round` works with two arguments, the limb's damage state will be 2. If that limb is damaged beyond the `max_damage`, at once, then `update_bodypart_damage_state` tries to update state higher than 2, preferably 3, as it should.

However, since you made the code block the update when damage is higher than `max_damage`, it will fail to update, and the state will be stuck at 2 when it should be 3.

I hope you understood my review better.

**Review by park66665 — APPROVED** [review]
All clear. This PR is good to go.

Each of damage is supposed to go up to double of `max_damage` on some cases.

**Pull request merged**
